components/vision.py

```diff
@@ -5,8 +5,9 @@
 import wpilib
 import wpiutil.log
 from magicbot import tunable, feedback
-from photonlibpy.photonCamera import PhotonCamera
-from photonlibpy.targeting.photonTrackedTarget import PhotonTrackedTarget
+from photonlibpy import PhotonCamera
+from photonlibpy.targeting import PhotonTrackedTarget
+
```

Comment:
nit: no additional arbitrary grouping of 3rd party imports
```suggestion
```

https://peps.python.org/pep-0008/#imports

Reply:
Checkmate: https://peps.python.org/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds